src/Connector.sol

```diff
@@ -78,6 +81,14 @@ contract CentrifugeConnector is Test {
         emit File(what, data);
     }
 
+    function file(bytes32 name, string memory domainName, uint32 domainId) public auth  {
```

Comment:
It seems uncommon to document Solidity code? For someone new to a codebase like this, I can't tell why we would need this function and where it would be used. Can we document that or explore that here in the comments?

Reply:
`file` is just a Maker standard name for an "updating parameters" method.